plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actions/CheckEdgeTargetResultAction.java

```diff
@@ -0,0 +1,40 @@
+/********************************************************************************
+ * Copyright (c) 2023 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+package org.eclipse.glsp.server.actions;
+
+/**
+ * {@link ResponseAction} for a {@link RequestCheckEdgeTargetAction}.
+ */
+public class CheckEdgeTargetResultAction extends ResponseAction {
+
+   public static final String KIND = "setTargetTypeHints";
```

Comment:
```suggestion
   public static final String KIND = "checkEdgeTargetResult";
```
Nees to match the KIND defiend in the client protocol